scalr/provider.go

```diff
@@ -50,14 +50,14 @@ func Provider() terraform.ResourceProvider {
 			"hostname": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: descriptions["hostname"],
+				Description: "Scalr instance hostname without scheme. Defaults to my.scalr.com.",
```

Comment:
```suggestion
				Description: "Scalr instance hostname without scheme. Defaults to  scalr.io.",
```
need to change default